Code/Tools/ProjectManager/Platform/Windows/ProjectBuilderWorker_windows.cpp

```diff
@@ -28,12 +28,13 @@ namespace O3DE::ProjectManager
     AZ::Outcome<QStringList, QString> ProjectBuilderWorker::ConstructCmakeBuildCommandArguments() const
     {
         QString targetBuildPath = QDir(m_projectInfo.m_path).filePath(ProjectBuildPathPostfix);
-        QString launcherTargetName = m_projectInfo.m_projectName + ".GameLauncher";
+        QString gameLauncherTargetName = m_projectInfo.m_projectName + ".GameLauncher";
+        QString serverLauncherTargetName = m_projectInfo.m_projectName + ".ServerLauncher";
```

Comment:
nit: can these QStrings be const?

Reply:
Fixed! 
When changing the string I noticed that Linux wasn't using the targetBuildPath string but Windows uses it for --build path. Are there any bugs on that? hah! Seems fine to me...

Reply:
Hmm, that unused targetBuildPath has been that way since the original implementation commit [cfc9ec2530dbb389a4e123db7d0448c8f22ccff9 ](https://github.com/o3de/o3de/commit/cfc9ec2530dbb389a4e123db7d0448c8f22ccff9)

Reply:
Ah, yes those unused `targetBuildPath` local variables should be removed, but doesn't have to be in this PR.